File: Simple_or_not.py
from math import sqrt
alpha = int(input('Введите число: '))
def is_simple(alpha):
	for i in range(2, int(sqrt(alpha)) + 1):
		if alpha % i == 0:
			print('Составное')
			break
	else:
		print('Простое')
is_simple(alpha)	
input('Press enter to exit')			

File: Simple_or_not.cpp
#include <iostream>
#include <cmath>
using namespace std;
bool is_simple(int alpha) {
	bool flag = true;
	for(int i = 2; i < sqrt(alpha) + 1; i++) {
		if (alpha % i == 0) {
			flag = false;
			break;
		}
	}
	return flag;
}
int main() {
	int alpha;
	cin>>alpha;
	if (is_simple(alpha)) {
		cout<<"Simple";
	}
	else {
		cout<<"Not simple";
	}
	return 0;
}